examples/react/file-upload/src/components/Loader.jsx

```diff
@@ -0,0 +1,24 @@
+function StatusLoader ({ progressStatus }) {
+  const { total, loaded, lengthComputable } = progressStatus
+  if (lengthComputable) {
+    const percentComplete = Math.floor((loaded / total) * 100)
+    return (
```

Comment:
Nit: Why not use native [progress element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) you could even style it as you’re pleased.

Reply:
oh wow I just literally did not know about this, wild! I'll take a quick pass at this - I suspect this is probably a better project for a future pass, but if it works pretty seamlessly I like the idea of modeling this

Reply:
ok I looked into this - it's cool and I'd like to use it in future work, but my attempts to style it didn't go as well as I'd hoped so I'm going to leave this one for future work, great suggestion tho